From bce5beafdc614df724cc954353f7e77b5064afbf Mon Sep 17 00:00:00 2001
From: Heinz Wrobel <Heinz.Wrobel@nxp.com>
Date: Tue, 12 May 2020 13:14:05 +0200
Subject: [PATCH] [PATCH] [LS1043A] Adjust device tree ranges for PCIe

- update PCI bus address to 0x80000000 for PCIe bus 2
- change AXI bus address to 0x4880000000 for PCIe bus 2
- increase address range to 0x80000000 for PCIe bus 2

Signed-off-by: Bogdan Folea <bogdan.folea@nxp.com>
Signed-off-by: Heinz Wrobel <Heinz.Wrobel@nxp.com>
---
 arch/arm64/boot/dts/freescale/fsl-ls1043a.dtsi | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm64/boot/dts/freescale/fsl-ls1043a.dtsi b/arch/arm64/boot/dts/freescale/fsl-ls1043a.dtsi
index e844edd85fda..4e038b2dcf09 100644
--- a/arch/arm64/boot/dts/freescale/fsl-ls1043a.dtsi
+++ b/arch/arm64/boot/dts/freescale/fsl-ls1043a.dtsi
@@ -832,7 +832,7 @@
 			iommu-map = <0 &smmu 0 1>;
 			bus-range = <0x0 0xff>;
 			ranges = <0x81000000 0x0 0x00000000 0x48 0x00010000 0x0 0x00010000   /* downstream I/O */
-				  0x82000000 0x0 0x40000000 0x48 0x40000000 0x0 0x40000000>; /* non-prefetchable memory */
+				  0x82000000 0x0 0x80000000 0x48 0x80000000 0x0 0x80000000>; /* non-prefetchable memory */
 			msi-parent = <&msi1>, <&msi2>, <&msi3>;
 			#interrupt-cells = <1>;
 			interrupt-map-mask = <0 0 0 7>;
-- 
2.17.1

